service/src/main/kotlin/fi/espoo/evaka/invoicing/service/InvoiceCorrection.kt

```diff
@@ -37,9 +35,8 @@ data class InvoiceCorrection(
     @Nested("invoice") val invoice: InvoiceWithCorrection?, // should later be a list?
 ) {
     fun toInvoiceRow() =
-        InvoiceRow(
-            id = InvoiceRowId(UUID.randomUUID()),
-            child = childId,
+        DraftInvoiceRow(
```

Comment:
should this function be called `toDraftInvoiceRow`?

Reply:
👍